src/components/PlayerSelect.js

```diff
@@ -2,7 +2,15 @@ import React from 'react'
 import injectSheet from 'react-jss'
 
 const styles = {
-
+	playerSelect: {
+		display: 'block',
+		marginBottom: 15,
+		border: '2px solid black',
+		borderRadius: 0,
+		fontSize: 12,
+		// appearance: 'none',
```

Comment:
delete commented code unless you think you'll need to come back to it later for some reason